aten/src/ATen/native/cuda/Blas.cpp

```diff
@@ -287,13 +287,13 @@ Tensor& addmm_out_cuda_impl(Tensor& result, const Tensor& self, const Tensor& ma
               self.const_data_ptr<scalar_t>(),
               result_->data_ptr<scalar_t>(),
               result_ld,
-#if 0
+#if defined(CUDA_VERSION) && CUDA_VERSION >= 11080
```

Comment:
for anyone looking at this in the future: we're using 11.8 here because that's what our CI has test coverage for